packages/strapi/lib/middlewares/parser/index.js

```diff
@@ -1,11 +1,49 @@
 'use strict';
 
 const body = require('koa-body');
-const qs = require('koa-qs');
+//const qs = require('koa-qs');
+var merge = require('merge-descriptors');
+const qs = require('qs');
 
 /**
  * Body parser hook
  */
+
+const addParser = app => {
+  merge(app.request, {
```

Comment:
Can you just use `Object.defineProperty` so we avoid adding a dependency for a simple usecase

Reply:
ok, done in latest commit.

However, looking at koa code, I'm wandering if is is actually needed: it seems they encapsulated koa-qs in their code altogether . See:
https://github.com/koajs/koa/blob/master/lib/request.js
Lines 172 and beyond